cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1044,6 +1044,15 @@
   </property>
 
   <property>
+    <name>log.publish.kafka.partition.key</name>
```

Comment:
Can we not having the name `kafka` in the config? As the computation of the partition has nothing to do with Kafka.